docs/ReleaseNotes.md

```diff
@@ -14,6 +14,7 @@ Bug Fixes:
 * Updated to IANA time zone data version 2019a
 * Fixed a bug where the DateRngFmt was not heeding the specified
   time zone when formatting the start and end dates
+* Fixed a bug which a default script for pa-IN should be Guru instead of Arabic
```

Comment:
Should this say "pa-IN" or "pa-PK" ?

Reply:
oh, yes. it is. As following  CLDR  `pa :pa-Gurn-IN` from `likelySubtags.json`.